server/api/knowledgeBase.ts

```diff
@@ -1318,9 +1201,10 @@ export const UploadFilesApi = async (c: Context) => {
           parentId: targetParentId,
           message:
             fileName !== originalFileName
-              ? `File uploaded as "${fileName}" (renamed to avoid duplicate)`
-              : "File uploaded successfully",
+              ? `File uploaded as "${fileName}" (renamed to avoid duplicate) - queued for processing`
+              : "File uploaded successfully - queued for processing",
           wasRenamed: fileName !== originalFileName,
+          uploadStatus: 'pending', // Indicate it's pending processing
```

Comment:
can we make enums out of uploadStatus and do the same for DB too instead of using string